src/tests/ftest/cart/rpc/cart_rpc_two_node.py

```diff
@@ -17,7 +17,8 @@ class CartRpcTwoNodeTest(CartTest):
     def test_cart_rpc(self):
         """Test CaRT RPC.
 
-        :avocado: tags=all,cart,pr,daily_regression,rpc,two_node
+        :avocado: tags=all,pr,full_regression
```

Comment:
why are we no longer including this in daily_regression? is that tag gone?

Reply:
`daily_regression` is still there, but we changed to `full_regression` per [this discussion](https://jira.hpdd.intel.com/browse/CORCI-1081?focusedCommentId=263916&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-263916) in gatekeeper channel ([CORCI-1081](https://jira.hpdd.intel.com/browse/CORCI-1081)).  The problem is this tag change prevents *all* cart tests from running in `daily_regression` runs, not just cart testing with Valgrind.  Since the CI resource throttling is done via the commit pragma (`Skip-func-test-vm-valgrind` defaults to `true`), the tag should be reverted to `daily_regression`.